modules/sequence/src/main/scala/lucuma/odb/sequence/gmos/longslit/Config.scala

```diff
@@ -162,6 +164,15 @@ object Config {
     override def defaultWavelengthDithers: List[WavelengthDither] =
       defaultWavelengthDithersNorth(this.grating)
 
+    val ccdMode: GmosCcdMode =
+      GmosCcdMode(
+        explicitXBin.getOrElse(defaultXBin),
+        explicitYBin.getOrElse(defaultYBin),
+        GmosAmpCount.Twelve,
+        explicitAmpGain.getOrElse(defaultAmpGain),
+        explicitAmpReadMode.getOrElse(defaultAmpReadMode)
+      )
```

Comment:
I think you can use the `Config` trait methods that do the explicit or default thing?

```scala
GmosCcdMode(
  xBin,
  yBin,
  GmosAmpCount.Twelve,
  ampGain,
  ampReadMode
)
```

Also, I think we could just define this up in `Config` and not have to duplicate it in `GmosSouth`?